pemFioi/quickAlgo/blockly_interface.js

```diff
@@ -469,7 +469,7 @@ function getBlocklyInterface(maxBlocks, nbTestCases) {
          this.loadPrograms();
       },
 
-      savePrograms: function() {
+      savePrograms: function(saveSubject) {
```

Comment:
savePrograms shouldn't have "saveSubject" as argument, it should be its responsibility to know whether to save the subject or not.